api/src/Entity/User.php

```diff
@@ -268,6 +270,23 @@ public function isEmployee(): bool
         return false;
     }
 
+    public function isAssociatedWithRepairer(int|string|Repairer $repairer): bool
+    {
+        if ($this->isBoss()) {
+            return !$this->repairers->filter(function (Repairer $repairerOfUser) use ($repairer) {
+                return $repairer instanceof Repairer ? $repairerOfUser->id === $repairer->id : $repairerOfUser->id == $repairer;
+            })->isEmpty();
+        }
+
+        if ($this->isEmployee() && $this->repairerEmployee) {
```

Comment:
```suggestion
        if ($this->repairerEmployee && $this->isEmployee()) {
```

But why not only checks `$this->isEmployee()`? In this context, if he is employed, it is suggested that he is also a repairer, right?